xikolo-ios/ViewControllers/Courses/CertificatesTableViewController.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  Created for xikolo-ios under MIT license.
+//  Copyright © HPI. All rights reserved.
+//
+
+import UIKit
+import SafariServices
+
+class CertificatesTableViewController: UITableViewController {
+
+    var course: Course!
+
+    // MARK: - Table view data source
+
+    override func numberOfSections(in tableView: UITableView) -> Int {
+        return 1
+    }
+
+    override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+        return section == 0 ? 3 : 0
```

Comment:
`return 3` would also be sufficient ;)
but you have to check if the certificate is available anyway